### PR TITLE
2180/vite to merge

### PR DESCRIPTION
**Pull request by JimmyRowland**

### CODE REVIEW

**Review by smattingly — CHANGES_REQUESTED** [review]
With npm's start script removed, I wasn't sure how to proceed. Please update the README with any new/changed instructions. Launched with vite script but hit this when I started typing address to create a farm:
image

**Review by smattingly — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -67,40 +70,38 @@ This approach runs the LiteFarm applications and database server directly on you
...
        ```sql
        CREATE DATABASE "pg-litefarm";
        CREATE DATABASE test_farm;
        exit;       
        ```
-   
-3. In a terminal, navigate to the root folder of the repository. Run `npx lerna bootstrap` to install the dependencies for both applications.
-   
+
+3. In a terminal, run `npm i` under the root folder of the repository, `npm i` under `packages/api`, and `pnpm i`
```
[the remark this answers is not in the document]
Comment (JimmyRowland) [reply README.md:83]: it's good enough for now. Added a new ticket to resolve package manager confusions.
[LF-2234]

[the remark this answers is not in the document]
Comment (smattingly) [reply README.md:83]: I will probably be responsible for automating deployment of this. I disagree that it's good enough for now.

[the remark this answers is not in the document]
Comment (smattingly) [reply README.md]: I made further comments on the Jira ticket requesting changes in dependency management.

File: README.md [hunk 2]
```diff
@@ -80,7 +80,8 @@ This approach runs the LiteFarm applications and database server directly on you
        exit;       
        ```
 
-3. In a terminal, run `npm i` under the root folder of the repository, `npm i` under `packages/api`, and `pnpm i`
+3. In a terminal, run `npm i pnpm -g` *if pnpm is not already installed*, run `npm i` under the root folder of the
```
Comment (smattingly) [inline README.md:83]: To ease automation, please put all dependencies in package.json or equivalent (https://12factor.net/dependencies). Please avoid global installs if not necessary.

**Review by smattingly — CHANGES_REQUESTED** [review]

**Pull request merged**